### DISCUSSION TITLE
Can you Deploy this into Replit?

### DISCUSSION THREAD

**Discussion by dragoon6714** [UseInterstellar/Interstellar#879] [Q&A] [answered]
i have a replit core subscription, so would i be able to deploy this repository to replit? I tried searching it up on google but there wasnt any clear answers for me. Thanks

**Comment by lakshinhemachandran** [comment]
I think you can
And Interstellar isn't on Google, I'm pretty sure.

**Reply by lakshinhemachandran** [reply]
Also before GitHub, they used Replit, but it's not free anymore, so they changed to GitHub.

**Reply by lakshinhemachandran** [reply]
So maybe it does work. but it's not free

**Reply by xdevnightless** [reply]
you can

**Comment by Aratox-Official** [comment, marked as answer]
I've tried doing so myself, although there are multiple errors with the code itself (I used 'Import from GitHub', unlike what I did with my last Interstellar fork). I'm receiving a [200, 599] Out of Range error, which I don't believe we can fix until the code itself is fixed. Seeing as how the project hasn't been worked on in a while at this point (the last modification was made 2 weeks ago), I think that'll be in 2025. **However, I can tell you that previous versions of "unsupported" Interstellar Repls do work in Deployments.**

**Reply by lakshinhemachandran** [reply]
The Out of Range error is showing up for many GitHub Interstellar users too. The simple fix is to make the port public, but I'm not sure if Replit has that function.

**Comment by dragoon6714** [comment]
> I've tried doing so myself, although there are multiple errors with the code itself (I used 'Import from GitHub', unlike what I did with my last Interstellar fork). I'm receiving a [200, 599] Out of Range error, which I don't believe we can fix until the code itself is fixed. Seeing as how the project hasn't been worked on in a while at this point (the last modification was made 2 weeks ago), I think that'll be in 2025. **However, I can tell you that previous versions of "unsupported" Interstellar Repls do work in Deployments.**

Thanks!
